src/tools/crossgen/crossgen.cpp

```diff
@@ -591,6 +596,10 @@ int _cdecl wmain(int argc, __in_ecount(argc) WCHAR **argv)
         {
             dwFlags |= NGENWORKER_FLAGS_READYTORUN;
         }
+        else if (MatchParameter(*argv, W("FragileNonVersionable")))
```

Comment:
Should we add this option to the usage output, or are we intentionally making it hard to discover?